src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsFeed.java

```diff
@@ -127,8 +129,11 @@ public Response postGtfsFeed(@FormParam("gtfsurl") String gtfsFeedUrl) {
         GtfsFeedModel gtfsFeed = (GtfsFeedModel) session.createQuery("FROM GtfsFeedModel "
                             + "WHERE gtfsUrl = '"+gtfsFeedUrl+"'").uniqueResult();
         Response.Status response = downloadGtfsFeed(saveFilePath, connection);
-        if(response == null) {
-            return generateError("Download Failed", "Downloading static GTFS feed from provided Url failed", Response.Status.BAD_REQUEST);
+        if (response == Response.Status.BAD_REQUEST) {
+            return generateError("Download Failed", "Downloading static GTFS feed from provided Url failed.", Response.Status.BAD_REQUEST);
+        }
```

Comment:
Minor formatting issue - can you please move the `else if...` to the same line as the preceding bracket?

So it would be:

`} else if (response == Response.Status.FORBIDDEN) {`

Same for other occurrences in this PR.